tests/sentry/api/endpoints/test_sentry_internal_app_token_details.py

```diff
@@ -60,16 +59,9 @@ def test_non_internal_app(self):
             slug=sentry_app.slug, organization=self.org, user=self.user
         )
 
-        client_id = install.sentry_app.application.client_id
```

Comment:
the following test files have cleanup bc the GrantExchange is handled by the `create_sentry_app_installation` factory